lms/templates/courseware/accordion.html

```diff
@@ -36,7 +36,7 @@ <h3 ${active_class} aria-label="${aria_label}">
               <p class="subtitle">${section['format']} ${due_date}</p>
 
               % if 'graded' in section and section['graded']:
-                  <img src="/static/images/graded.png" alt="Graded Section">
+              <i class="icon fa fa-pencil-square-o" aria-hidden="true" data-tooltip="${_("This section is graded.")}"></i>
```

Comment:
We talked about this in person. I think the tentative plan is to move all of our <i> elements to be svgs with the PL work, so I've left this in for now. Depending on the timeline for PL use in the LMS though, we could decide to revisit this and have a PR that moves all <i> tags over to be <spans> (ensuring all tests still pass, and removing element based styles currently using the <i> element :/ )